src/backend/distributed/planner/multi_explain.c

```diff
@@ -199,6 +200,17 @@ CitusExplainScan(CustomScanState *node, List *ancestors, struct ExplainState *es
 		return;
 	}
 
+	if (AlterTableInProgress())
+	{
+		/*
+		 * EXPLAIN for ALTER TABLE commands are not supported.
+		 */
+		ExplainPropertyText("Citus Explain Scan",
+							"Explain for ALTER TABLE commands are not supported",
```

Comment:
minor: EXPLAIN for ALTER table is not supported Postgres anyway. 

Maybe change the message to something like:

Explain for constraint validation queries during ALTER TABLE commands are not supported by Citus